index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/fontawesome.min.css" integrity="sha512-SgaqKKxJDQ/tAUAAXzvxZz33rmn7leYDYfBP+YoMRSENhf3zJyx3SBASt/OfeQwBHA1nxMis7mM3EV/oYT6Fdw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="index.css">
+    <title>HennaHub</title>
+</head>
+<body>
+    <header>
+     <i class="fa-solid fa-bars" aria-hidden="true"></i>
+        <div class="top-header">
+            <ul>
+                <li><a href="https://web.facebook.com/hennahubuk/?_rdc=1&_rdr"><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <div class="sub-header">
+            <a href="https://hennahubuk.business.site/"><img src="./images/logo.jfif" alt="website_logo"></a>
+      <div class="header-right">
+        <nav>
+          <ul class="links">
+            <li><a class="nav-link active" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#">Program</a></li>
+            <li><a class="nav-link" href="#">Join</a></li>
+            <li><a class="nav-link" href="#">Sponsor</a></li>
+            <i class="fa-solid fa-xmark"></i>
+          </ul>
+          <Span>Special Seat</Span>
+        </nav>
+        </div>
+    </header>
+    <div class="begin">
+        <div class="introduction">
+            <div class="intro-heading">
+                <p>"Hello HennaHub Globe!"</p>
+                <h1 class="intro-trans-anim">HENNAHUBUK'S CREATIVE DESIGN APPRENTICE SUMMIT <br>2023</h1>
+            </div>
+            <p class="intro-text">A joyful celebration of creativity and mehndi design for the first apprentice graduation og HennaHubUK academy</p>
+            <h2>2023-12-20(WED) ~ 21(THU)</h2>
+            <span>@ 1312 Gallowgate Glasgow G31 4DR UK</span>
+        </div>
+    </div>
```

Comment:
- [ ] Nice work implementing the headline section, it looks almost identical to the provided design :clap: However It represents a generic standalone section of the document, so it would be nice to wrap it in a `section` tag for semantics reasons. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.